plugins/automem/scripts/process-session-memory.py

```
@@ -0,0 +1,512 @@
+            "minimum_changes": 3,  # Minimum file changes for significance
+            "minimum_lines": 10,  # Minimum lines changed
+        }
+        
+        if filters_path.exists():
+            try:
+                with open(filters_path, 'r') as f:
+                    loaded = json.load(f)
+                for key, value in default_filters.items():
+                    if key not in loaded:
+                        loaded[key] = value
+                return loaded
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)